Pack3/games/AwShirt/content/AwShirtSlogans.jet

```diff
@@ -1182,7 +1182,7 @@
         },
         {
             "id": 41225,
-            "suggestion": "[KE RALPOLENG] Aw nertz",
+            "suggestion": "Txavä' nang",
```

Comment:
Don't know the original meaning hrh

Reply:
Agree, it could be less confusing because I google the meaning and this was a bit literal translation of it. (at least one version of it)